app/ai/voice/agents/breeze_buddy/mcp/__init__.py

```
@@ -218,3 +228,142 @@ async def get_mcp_global_functions(
+        return self._functions
+
+    async def open(self) -> "MCPSessionPool":
+        for server in self._mcp_config.servers:
+            if not server.enabled:
+                continue
+            label = server.name or server.url
+            try:
+                client, funcs = await self._open_server(server)
+                self._clients.append(client)
+                self._functions.extend(funcs)
+            except BaseException as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `BaseException`

(BLE001)